plugins/android-extensions/android-extensions-compiler/src/org/jetbrains/kotlin/android/parcel/ir/AndroidSymbols.kt

```diff
@@ -0,0 +1,477 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+// This file was autogenerated based on android.jar, do not edit it directly.
```

Comment:
Personal nit, but I feel it useful to have a reference to the script that generated this file, too.

Reply:
I hear you. I'm just putting this off for later, since the script in question is more than a little bit messy and larger than it should be. That said, I would also like to include it at some point.